contracts/deploy/base/009_upgrade_vault.js

```diff
@@ -0,0 +1,77 @@
+const { deployOnBaseWithGuardian } = require("../../utils/deploy-l2");
+const { deployWithConfirmation } = require("../../utils/deploy");
+const addresses = require("../../utils/addresses");
+const { utils } = require("ethers");
+
+module.exports = deployOnBaseWithGuardian(
+  {
+    deployName: "009_upgrade_vault",
+  },
+  async ({ ethers }) => {
+    const cOETHbVaultProxy = await ethers.getContract("OETHBaseVaultProxy");
+    const cOETHbVault = await ethers.getContractAt(
+      "IVault",
+      cOETHbVaultProxy.address
+    );
+    const cOETHbDripperProxy = await ethers.getContract("OETHBaseDripperProxy");
+
+    // Deploy new implementation
+    const dOETHbVaultCore = await deployWithConfirmation("OETHBaseVaultCore", [
+      addresses.base.WETH,
+    ]);
+    const dOETHbVaultAdmin = await deployWithConfirmation("OETHBaseVaultAdmin");
+
+    return {
+      actions: [
+        {
+          // 1. Upgrade VaultCore
+          contract: cOETHbVaultProxy,
+          signature: "upgradeTo(address)",
+          args: [dOETHbVaultCore.address],
+        },
+        {
+          // 2. Upgrade VaultAdmin
+          contract: cOETHbVault,
+          signature: "setAdminImpl(address)",
+          args: [dOETHbVaultAdmin.address],
+        },
+        {
+          // 3. Set allocate threshold
+          contract: cOETHbVault,
+          signature: "setAutoAllocateThreshold(uint256)",
+          args: [utils.parseEther("10")], // 10 OETH
```

Comment:
currently on the base mainnet contract this is set to 25k. No auto allocate will be happening on base Afaik?

Reply:
I don't think it's 25k. Thats like more than half of OETH supply. It's 10 OETH on mainnet as well. But this is probably gonna be unused since we won't have a default strategy

Reply:
<img width="475" alt="Screenshot 2024-08-28 at 10 55 15" src="https://github.com/user-attachments/assets/5cc893dd-320a-420a-b533-d920f84ac348">
![Screenshot 2024-08-28 at 10 55 11](https://github.com/user-attachments/assets/b3e073c7-299d-4471-93bf-a2a0123da934)

Looks like it is 25k